### ISSUE TITLE
better error formatting using ariadne?

### ISSUE THREAD

**Issue by youknowone** [RustPython/RustPython#4960]
https://docs.rs/ariadne/latest/ariadne/

**Comment by DimitrisJim** [comment]
I don't believe there's is any PEP that might define what exception output/formatting might look like so we should be free to choose. If ariadne can be plugged in easily it could be something to think about.
